src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -238,19 +238,33 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig w
     * we should evaluate how to handle failed processing
     */
   private[this] def processTezosAccounts(): Future[Done] = {
+    import cats.instances.list._
```

Comment:
Why not put this above object definition?

Reply:
This might change, but originally cats was used only within a limited scope of the code.

I tried to limit cats values to local scopes, as long as usage is not wide-spread... maybe it's the right time to, I didn't stop to reconsider if it makes sense